doc/ref_model/chapters/chapter03.md

```diff
@@ -13,21 +12,21 @@
 * [3.4 Physical Resources](#3.4)
 * [3.5 Network](#3.5)
 
-There is the necessity to clearly define which kind of infrastructure resources a shared network function virtualisation infrastructure (NFVI) will provide for hosting workloads including virtual network functions (VNFs) and/or cloud-native network functions (CNF), so that the requirements of the workloads match the capabilities of the NFVI.
+There is the necessity to clearly define which kind of infrastructure resources a shared cloud infrastructure (network function virtualisation infrastructure, NFVI) will provide for hosting workloads including virtual network functions (VNFs) and/or cloud-native network functions (CNF), so that the requirements of the workloads match the capabilities of the cloud infrastructure.
```

Comment:
Not sure what is being conveyed here including the beginning.  Are we defining the infrastructure resources that the cloud infrastructure will provide to the workloads? If yes, then the workloads must utilise the capabilities offered -- how would that lead to ("so that") "requirements of the workloads match the capabilities."  The following is a suggestion but still not satisfactory.
```suggestion
It is necessary to clearly define the infrastructure resources and their capabilities a shared cloud infrastructure (network function virtualisation infrastructure, NFVI) will provide for hosting workloads including virtual network functions (VNFs) and/or cloud-native network functions (CNFs).
```

Reply:
@pgoyal01, I agree.  This was a left-over form the original text. I will commit the suggestion and I will suggest you approving this PR.  We can polish up some of this later on.